src/export.js

```diff
@@ -116,6 +120,15 @@ function exportDataset(opts) {
       miss.through(reportDocumentCount),
     )
 
+    jsonStream.on('end', () => {
+      debug('Export stream completed')
+    })
+
+    jsonStream.on('error', (streamErr) => {
+      debug('Export stream error: ', streamErr)
+      reject(streamErr)
+    })
+
     miss.finished(jsonStream, async (err) => {
       if (err) {
         return
```

Comment:
We can potentially reject the promise here as well.

Reply:
Added by Espen here: https://github.com/sanity-io/export/pull/4/files#diff-94aecbeb3eaa9808869af24b7c12dca7d1d85c93df0ee7898a3727454d600abfR131

So, I guess I can remove the rejection above.